### PR TITLE
Make Client.closing() idempotent

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: neuromation/api/client.py [hunk 1]
```diff
@@ -50,6 +51,9 @@ def __init__(
         self._images: Optional[Images] = None
 
     async def close(self) -> None:
+        if self._closing:
+            return
+        self._closing = True
         await self._core.close()
```
Comment (serhiy-storchaka) [inline neuromation/api/client.py]: If any of operation will fail, would not client keep some unclosed resources? Is it worth to guarantee closing all resources?
```python
    try:
        close_resource1()
    finally:
        try:
            close_resource2()
        finally:
            close_resource3()
```
or
```python
    try:
        try:
            close_resource1()
        finally:
            close_resource2()
    finally:
        close_resource3()
```

File: neuromation/api/client.py [hunk 2]
```diff
@@ -27,6 +27,7 @@ def __init__(
         *,
         timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT
     ) -> None:
+        self._closing = False
```
[the remark this answers is not in the document]
Comment (asvetlov) [reply neuromation/api/client.py]: I not sure.
My initial motivation for `_closing` was that after setting `self._closing =  True` the client is not *closed* actually but in *initialized closing process* phase.
asyncio uses *-ing* flags for similar state, e.g. `transport.is_closing()` .

Honestly I don't care too much, I can rename the (*private*) attr if you insist.

File: neuromation/api/client.py (hunk 1 above)
Comment (asvetlov) [reply neuromation/api/client.py]: Good question.
aiohttp closes client session and open socket objects on GC but does it in *dirty style*: with warnings etc.

I think it is enough, at least until people start objecting.

File: neuromation/api/client.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (serhiy-storchaka) [reply neuromation/api/client.py]: I do not insist. It just looked slightly unusual to me. In other contexts (files and file-like objects) the `closed` attribute is more common and may be public.

**Pull request merged**